src/main/java/org/cardanofoundation/job/repository/ledgersyncagg/LatestTokenBalanceRepository.java

```diff
@@ -27,6 +27,15 @@ select max(ltb.block_time) from latest_token_balance ltb
       nativeQuery = true)
   Long getTheSecondLastBlockTime();
 
+  @Query(
+      value =
+          """
+                  select max(ltb.slot) from latest_token_balance ltb
```

Comment:
Just a tip: From my point of view writing SQL Command in upper case makes it easier reading the whole command.